core/arch/arm/plat-mediatek/main.c

```diff
@@ -12,8 +12,10 @@
 #include <platform_config.h>
 #include <stdint.h>
 
+#if defined(CFG_CONSOLE_ENABLE)
```

Comment:
`#if (CFG_TEE_CORE_LOG_LEVEL != 0)`

Reply:
Yes much better thanks for the suggestion